util/parse.cpp

```diff
@@ -20,21 +20,23 @@ Params ParseArgs(int argc, char** argv) {
 
 class NullEventHandler : public YAML::EventHandler {
  public:
-  virtual void OnDocumentStart(const YAML::Mark&) {}
-  virtual void OnDocumentEnd() {}
+  virtual void OnDocumentStart(const YAML::Mark&) override {}
```

Comment:
Can you also remove the "virtual"s, which is redundant with "override"?

Reply:
Done. (Personally, I like to keep both, which is why I didn't do that initially, but I'm fine with your preference.)

BTW, I notice belatedly that this class does not seem to be used?? (Without this change, it is not instantiable...)

Reply:
Ha, that's pretty funny. I think I at one point used this as a way to measure speed of the event parsing, and never got rid of it.